doc/contributing.md

```diff
@@ -1,53 +0,0 @@
-# Contribution guidelines
-
-This document sets out the methods and practices for contributing to the Verovio project.
-
-## Contributor License Agreement
-
-In order to contribute the the project you need to sign and send us a [Contributor License Agreement](https://rism.digital/resources/verovio-cla.pdf) (CLA). However, following a generally accepted practice, very small contributions (below 20 lines of code) will be accepted without signing the CLA.
-
-## How to contribute
```

Comment:
Where is this section in the new re-organization?

Reply:
Removed, following https://github.com/rism-digital/verovio/pull/2923#issuecomment-1153614305

Reply:
we should have a minimal with a link to https://book.verovio.org/contributing/guidelines.html

Reply:
(actually we have in in `.github`, which looks weird but seems to be fine)

Reply:
As this file is a GitHub thing I thought it would be best to put it there with the issue templates, following
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors